commonui/src/test/java/com/hyperwallet/android/ui/common/util/CurrencyParserTest.java

```diff
@@ -90,12 +90,27 @@ public void testAllCurrencyFormats() {
         }
     }
 
+    @Test
+    public void testAllCurrencyFormatsWithSymbol() {
+        String amount = "1000000";
+            Context context = ApplicationProvider.getApplicationContext();
+            String currency = CurrencyParser.getInstance(context).formatCurrencyWithSymbol("TWD", amount);
+            assertThat(currency, is("NT$1,000,000"));
+    }
     @Test
     public void testGetNumberOfFractionDigits_ThreeDigitDecimal(){
         Context context = ApplicationProvider.getApplicationContext();
         int noOfDigits = CurrencyParser.getInstance(context).getNumberOfFractionDigits("TND");
         assertThat(noOfDigits, is(3));
     }
+
+    @Test
+    public void testGetCurrency() {
+        Context context = ApplicationProvider.getApplicationContext();
+        CurrencyDetails currency = CurrencyParser.getInstance(context).getCurrency("TND");
```

Comment:
var name should bee `currencyDetails`

Reply:
It is not variable, it is a  class name.

Reply:
yes I know, I am asking you to rename the variable 
`CurrencyDetails currency =`
Change to 
`CurrencyDetails currencyDetails =`

Reply:
Done